src/components/buttonPanel.jsx

```diff
@@ -10,33 +11,33 @@ class ButtonPanel extends Component {
   render() {
     return (
       <div>
-        <div>
-          <Button name="AC" />
-          <Button name="+/-" />
-          <Button name="%" />
+        <div className="row">
+          <Button name="AC" color="gray" />
+          <Button name="+/-" color="gray" />
+          <Button name="%" color="gray" />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.

Reply:
Thank you for this suggestion. I pointed that out [here](https://github.com/johnsonsirv/mvp-react-calculator/pull/3)as something I will still implement later.